lib/player_widget.dart

```diff
@@ -202,14 +212,14 @@ class _PlayerWidgetState extends State<PlayerWidget> {
     _audioPlayer.onPlayerStateChanged.listen((state) {
       if (mounted) {
         setState(() {
-          _audioPlayerState = state;
+          _playerState = state;
         });
       }
     });
 
     _audioPlayer.onNotificationPlayerStateChanged.listen((state) {
       if (mounted) {
-        setState(() => _audioPlayerState = state);
+        setState(() => _playerState = state);
       }
     });
 
```

Comment:
remove _playingRouteState below if this variable is no longer used

Reply:
yeah noticed that this morning too :)